sites/fast-component-explorer/build/convert-readme.js

```diff
@@ -8,6 +8,9 @@ const fs = require("fs");
 const MarkdownIt = require("markdown-it");
 
 const srcDir = "../../packages/web-components/fast-components/src/**/README.md";
+// TODO: update this when the repository is renamed
```

Comment:
We should add issue number with this todo and perhaps include this to the bigger list => https://github.com/microsoft/fast-dna/issues/3307

Reply:
You left a similar comment which I addressed here: https://github.com/microsoft/fast-dna/pull/3325#discussion_r441218858